packages/devtools_app/lib/src/logging/logging_screen.dart

```diff
@@ -30,12 +29,7 @@ class LoggingScreen extends Screen {
   String get docPageId => screenId;
 
   @override
-  Widget build(BuildContext context) {
-    return !(serviceManager.connectedApp.isFlutterWebAppNow &&
-            serviceManager.connectedApp.isProfileBuildNow)
```

Comment:
I'm struggling to remember why this was disabled for flutter web profile builds to begin with. @jacob314 @devoncarew do either of you recall?

Reply:
Looks like this is something our current flags don't allow us to represent. For now I've committed the test with it enabled for Profile Flutter Web apps and a TODO.

Reply:
it is reasonable to expose in profile builds. Just check it doesn't throw exceptions. The structured error handling won't work in profile builds.

Reply:
How can I get the VM service address for a web profile build to test? It doesn't seem to be provided from the terminal (using `flutter run -d chrome --profile`) nor in the JSON in `--machine` mode as far as I can see.

Reply:
Turns out you can't connect DevTools to web profile mode as there's no VM Service. So this existing check didn't do anything. I've removed the corresponding test.